app/scripts/components/navigatorComponents/listItems.jsx

```diff
@@ -239,9 +238,11 @@ class ListItems extends React.Component {
     tableRowColumnMenu(child, menuWidth, projectRoles) {
         let optionsMenu
         if (child.kind === Kind.DDS_FILE) {
-            optionsMenu = <FileOptionsMenu {...this.props} clickHandler={()=>this.setSelectedEntity(child.id, Path.FILE, true)}/>
+            let role = projectRoles.get(child.project.id);
+            optionsMenu = role && role === 'Project Admin' && <FileOptionsMenu {...this.props} clickHandler={()=>this.setSelectedEntity(child.id, Path.FILE, true)}/>
         } else if (child.kind === Kind.DDS_FOLDER) {
-            optionsMenu = <FolderOptionsMenu {...this.props} clickHandler={()=>this.setSelectedEntity(child.id, Path.FOLDER, true)}/>
+            let role = projectRoles.get(child.project.id);
+            optionsMenu = role && role === 'Project Admin' && <FolderOptionsMenu {...this.props} clickHandler={()=>this.setSelectedEntity(child.id, Path.FOLDER, true)}/>
```

Comment:
Same here. Although some roles shouldn't be able to see the folder options menu. 
For folder menu it should be something like `role !== Roles.project_viewer && role !== Roles.file_uploader && role !== Roles.file_downloader;`

Reply:
👍